packages/js/product-editor/src/components/variations-table/variation-actions-menu/variation-actions-menu.tsx

```diff
@@ -0,0 +1,291 @@
+											) }
+										</MenuItem>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt(
+													'regular_price',
+													__(
+														'Enter a value (fixed or %)',
+														'woocommerce'
+													),
+													( value ) =>
+														addFixedOrPercentage(
+															variation.regular_price,
+															value
+														)?.toFixed( 2 )
+												);
+												onClose();
+											} }
+										>
+											{ __(
+												'Increase list price',
+												'woocommerce'
+											) }
+										</MenuItem>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt(
+													'regular_price',
+													__(
+														'Enter a value (fixed or %)',
+														'woocommerce'
+													),
+													( value ) =>
+														addFixedOrPercentage(
+															variation.regular_price,
+															value,
+															-1
+														)?.toFixed( 2 )
+												);
+												onClose();
+											} }
+										>
+											{ __(
+												'Decrease list price',
+												'woocommerce'
+											) }
+										</MenuItem>
+									</MenuGroup>
+									<MenuGroup
+										label={ __(
+											'Sale price',
+											'woocommerce'
+										) }
+									>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt( 'sale_price' );
+												onClose();
+											} }
+										>
+											{ __(
+												'Set sale price',
+												'woocommerce'
+											) }
+										</MenuItem>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt(
+													'sale_price',
+													__(
+														'Enter a value (fixed or %)',
+														'woocommerce'
+													),
+													( value ) =>
+														addFixedOrPercentage(
+															variation.sale_price,
+															value
+														)?.toFixed( 2 )
+												);
+												onClose();
+											} }
+										>
+											{ __(
+												'Increase sale price',
+												'woocommerce'
+											) }
+										</MenuItem>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt(
+													'sale_price',
+													__(
+														'Enter a value (fixed or %)',
+														'woocommerce'
+													),
+													( value ) =>
+														addFixedOrPercentage(
+															variation.sale_price,
+															value,
+															-1
+														)?.toFixed( 2 )
+												);
+												onClose();
+											} }
+										>
+											{ __(
+												'Decrease sale price',
+												'woocommerce'
+											) }
+										</MenuItem>
+										<MenuItem
+											onClick={ () => {
+												handlePrompt(
+													'date_on_sale_from_gmt',
+													__(
+														'Sale start date (YYYY-MM-DD format or leave blank)',
+														'woocommerce'
+													)
+												);
```

Comment:
I think we should also ask for the Sale end date here, same as the old UI.
Also should we disable this if no sale price has been set?

Reply:
Ohh, right!, let me add the the `sale end date` too.